tailmac/LICENSE/LICENSE.txt

```diff
@@ -0,0 +1,8 @@
+Copyright © 2023 Apple Inc.
```

Comment:
Probably not 😛

Reply:
Ha... There's still a few bits of code from the virtualization sample app in there so I retained this per the terms.